.github/workflows/contracts-ecdsa.yml

```diff
@@ -203,4 +222,63 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: npm publish --access=public --tag ${{ github.event.inputs.environment }} --network=${{ github.event.inputs.environment }}
 
-      # TODO: Notify CI about completion of the workflow
+      - name: Notify CI about completion of the workflow
+        uses: keep-network/ci/actions/notify-workflow-completed@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
+        with:
+          module: "github.com/keep-network/keep-core/ecdsa"
+          url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
+          environment: ${{ github.event.inputs.environment }}
+          upstream_builds: ${{ github.event.inputs.upstream_builds }}
+          upstream_ref: ${{ github.event.inputs.upstream_ref }}
+          version: ${{ steps.npm-version-bump.outputs.version }}
+
+      - name: Upload files needed for etherscan verification
+        uses: actions/upload-artifact@v2
+        with:
+          name: Artifacts for etherscan verifcation
+          path: |
+            ./solidity/ecdsa/deployments
+            ./solidity/ecdsa/package.json
+            ./solidity/ecdsa/yarn.lock
+
+  contracts-etherscan-verification:
```

Comment:
We could use [resuable workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows) for this job. See the example in https://github.com/keep-network/keep-core/pull/3132. But let's not block this PR on that.